### PR TITLE
[15.0][MIG] account_check_printing_report_dlt103

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests

File: account_check_printing_report_dlt103/__manifest__.py [hunk 1]
```diff
@@ -16,7 +15,6 @@
     "data": [
         "data/report_paperformat.xml",
         "data/report_paperformat_parameter.xml",
-        "data/account_payment_check_report_data.xml",
```
Comment (MiquelRForgeFlow) [inline account_check_printing_report_dlt103/__manifest__.py]: Why did you delete this?

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: account_check_printing_report_dlt103/__manifest__.py (hunk 1 above)
Comment (JasminSForgeFlow) [reply account_check_printing_report_dlt103/__manifest__.py]: That file is used to add option dlt103 for check printing, which is handled with selection field account_check_printing_layout on payment.journal in v15, so no longer a need for that file with that declaration way
